### DISCUSSION TITLE
Switching function

### DISCUSSION THREAD

**Discussion by PabloPiaggi** [deepmodeling/deepmd-kit#2446] [Q&A] [answered]
Hello,

I see in the documentation of the se_a.py descriptor ([this line](https://github.com/deepmodeling/deepmd-kit/blob/3ac8c4c77b61ee4c645cfb59eaefb104d73c0309/deepmd/descriptor/se_a.py#L89)) that the switching function is a fifth order polynomial instead of the cosine function described in the DeepPotential-SE [paper](https://proceedings.neurips.cc/paper/2018/hash/e2ad76f2326fbc6b56a45a56c59fafdb-Abstract.html). I used the DeePMD-kit versions v2.0.0.b2 and v2.1.3 in my work. Which switching function was used by default in each case? I tried to read the code to see which function was used, but I wasn't successful. Thank you.

Regards,

Pablo

**Comment by njzjz** [comment, marked as answer]
The switching function is a fifth order polynomial since b688b6b8ce9b56ea44ed4d954b10da737e5d4c56 (v0.12.0).

We will have [a new preprint](https://doi.org/10.48550/arXiv.2304.09409) next week, which should be followed instead of the old paper.

**Reply by PabloPiaggi** [reply]
Thanks, @njzjz, for the clear answer.
